File: 3069-distribute-elements-into-two-arrays-i/3069-distribute-elements-into-two-arrays-i.kt
class Solution {
    fun resultArray(nums: IntArray): IntArray {
        val a = mutableListOf<Int>()
        val b = mutableListOf<Int>()
        a.add(nums[0])
        b.add(nums[1])
        for(i in 2 until nums.size){
            if(a.last() > b.last()){
                a.add(nums[i])
            }
            else{
                b.add(nums[i])
            }
        }
        val res = a + b
        return res.toIntArray()
    }
}

File: 3069-distribute-elements-into-two-arrays-i/3069-distribute-elements-into-two-arrays-i.py
class Solution(object):
    def resultArray(self, nums):
        l1,l2 = [],[]
        l1.append(nums[0])
        l2.append(nums[1])
        for i in range(2,len(nums)):
            if l1[-1] > l2[-1]:
                l1.append(nums[i])
            else:
                l2.append(nums[i])
        res = l1 + l2
        return res

        